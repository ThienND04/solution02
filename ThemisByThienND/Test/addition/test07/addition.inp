10
8000 10000 4000 700000000
1000 3000 5000 700000000
4000 10000 1000 1000000000
3000 3000 8000 1000000000
8000 10000 1000 900000000
4000 10000 10000 1000000000
1000 4000 10000 500000000
7000 6000 8000 700000000
3000 8000 1000 800000000
10000 10000 10000 800000000
