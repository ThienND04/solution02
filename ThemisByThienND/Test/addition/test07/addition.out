1531320001
16333543334
12500200001
6944430555
5062612501
2500100001
3125100001
729195834
13333533334
3200120001
